TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/core/objects/Proxy.java

```diff
@@ -295,12 +296,12 @@ public boolean isConnected() {
         return connected;
     }
 
-    public DnsRecord getDnsRecord() {
-        return dnsRecord;
+    public ArrayList<DnsRecord> getDnsRecord() {
+        return dnsRecords;
     }
 
-    public void setDnsRecord(DnsRecord dnsRecord) {
-        this.dnsRecord = dnsRecord;
+    public void addDnsRecord(DnsRecord dnsRecord) {
+        this.dnsRecords.add(dnsRecord);
         //TODO Work for Timo. Object "DnsRecord" does not exist in TimoCloudAPI Project
```

Comment:
Please remove the TODO comment and either implement the DnsRecordObject in the API or open an issue for that